src/wallet/rpcwallet.cpp

```diff
@@ -929,11 +926,6 @@ static UniValue sendmany(const JSONRPCRequest& request)
 
     EnsureWalletIsUnlocked(pwallet);
 
-    // Check funds
-    if (totalAmount > pwallet->GetLegacyBalance(ISMINE_SPENDABLE, nMinDepth)) {
```

Comment:
Strictly speaking it is changing behavior, since it might throw a different rpc error (or none at all and succeed) now. However, the previous behavior was not well specified by the documentation string of `minconf`. And my attempt at making sense of it failed: 
*  rpc: Actually use sendmany::minconf #15595

Reply:
By not throwing an error the client may think `min_depth` is still used and never update his code.

> Also, maybe this should raise an error if min_depth is passed and is set to anything other than 1.

I think @ryanofsky suggestion should be considered.

Reply:
That is just going to open a can of worms. I think, either it should throw for any value (including 1, the default), since that was ignored before as well, or never throw at all.

If it throws for all values, you might as well just remove the parameter and break the interface. Though, then that'd have to go through a `-deprecatedrpc` cycle. I don't think this effort is justified. Please explain why ignoring this parameter could lead to any meaningful issues

Reply:
I think ignoring doesn't raise awareness and clients remain deluded, as we were.

Reply:
I also understand the effort concern, but we do have a clear way to deprecate things and break backward compatibility, so we deal with this as a special case, and leave the parameter sitting there?

Reply:
I can see both sides. In this case though, I think we should just silently ignore. I don't see any downside to clients continuing to specify this and having it ignored. The help text has been updated to "ignore dummy value", so I don't think there's any chance of users being confused by this.

Enforcing that this be set to 1 would give the impression that the passed value actually does something.

Reply:
The help text is worthless for existing software.

Reply:
> The help text is worthless for existing software.

For existing software, the only thing throwing an error would do is make the user change the client so it sets this to 1 or omit it. The end result will be the same as if the RPC doesn't throw an error, except there would be more disruption for users during transition.

If this PR were actually deprecating useful functionality, then I'd agree with you, but it's not. This option has never done what the help text implies that it should do.